Shared/Samples/Geocode offline/GeocodeOfflineView.Model.swift

```diff
@@ -0,0 +1,206 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension GeocodeOfflineView {
+    /// The view model for the sample.
+    @MainActor
+    class Model: ObservableObject {
+        // MARK: Properties
+        
+        /// A map with a tiled layer of the streets in San Diego, CA, USA.
+        let map = {
+            let map = Map()
+            
+            // Create a tiled layer using the local tile package.
+            let tileCache = TileCache(fileURL: .streetMap)
+            let tiledLayer = ArcGISTiledLayer(tileCache: tileCache)
+            
+            // Add the layer to the map.
+            map.addOperationalLayer(tiledLayer)
+            
+            return map
+        }()
+        
+        /// The graphics overlay for the marker graphic.
+        let graphicsOverlay = GraphicsOverlay()
+        
+        /// The red map marker graphic used to indicate a given location on the map.
+        private let markerGraphic = {
+            // Create a symbol using the image from the project assets.
+            guard let markerImage = UIImage(named: "RedMarker") else { return Graphic() }
+            let markerSymbol = PictureMarkerSymbol(image: markerImage)
+            
+            // Change the symbol's offsets, so it aligns properly to a given point.
+            markerSymbol.leaderOffsetY = markerImage.size.height / 2
+            markerSymbol.offsetY = markerImage.size.height / 2
+            
+            // Create a graphic with the symbol.
+            return Graphic(symbol: markerSymbol)
+        }()
+    
+        /// The value used to offset the callout when the map magnifier is showing.
+        private let magnifierOffset = {
+            // Get the magnifier image from ArcGIS package bundle.
+            let bundle = Bundle(identifier: "com.esri.ArcGIS") ?? Bundle(identifier: "ArcGIS")
+            let image = UIImage(named: "Magnifier", in: bundle, with: nil) ?? UIImage()
+            
+            // Determine the amount by which we will need to offset the callout along y-axis.
+            return CGPoint(x: .zero, y: -image.size.height)
+        }()
```

Comment:
While this might improve user experience, I hesitate to include it as it provides too much unneeded details, even if it is in the old implementation.
Besides, this offset is currently (0, 0) as the image isn't correctly accessed from the local Swift package for some reason. I tried using image literal `#imageLiteral(resourceName: "Magnifier")` and it doesn't work either.

So my suggestion is to get rid of the offset altogether.